example/Switches.jsx

```diff
@@ -38,6 +38,11 @@ export default function Switches () {
         <Switch design='checkbox'>This is a toggle switch with checkbox design</Switch>
       </Code>
 
+      <h5>Checkbox with legal size text</h5>
+      <Code>
+        <Switch design='checkbox' legal>This is a toggle switch with checkbox design and legal size text that usually will fold into multiple lines</Switch>
```

Comment:
Now we have two ways of defining whether something should be of "legal" design: `<Paragraph design="legal" />` and `<Switch design="checkbox" legal />`. Perhaps we need a convention?